04/index.html

```diff
@@ -1,27 +1,35 @@
+<!-- Site published https://krzysiekswol.github.io/practice-html-and-css-rwd/ -->
 <!DOCTYPE html>
 <html lang="pl">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./styles/global.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(min-width: 768px)" href="./styles/desktop.css">
+
+
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
 </head>
 <body>
     
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
+    <nav class="nav">
+        <label for="menu--switcher" class="menu"></label>
+        <input type="checkbox" id="menu--switcher">
+        <ul class="list">
+            <li class="list__item">
+                <a class="link" href="#">start</a>
```

Comment:
Jeśli ma to być zgodne z BEM to raczej `list__link` raczej sam link nie będzie osobnym blokiem, ale oczywiście do za argumentowania :)

Tutaj znajdziesz częste błędy dot. BEM, jakie popełniają mentee: 
https://devmentor.pl/b/metodologia-bem-w-css-i-sass